### DISCUSSION TITLE
For saving the storage space

### DISCUSSION THREAD

**Discussion by nothingness6** [Acly/krita-ai-diffusion#563] [Q&A]
I've already installed ComfyUI through Stability Matrix, so I don't want to waste storage space. In this case, how can I manage it?

**Comment by ETCore7** [comment]
When Comfy is running you can connect to the local URL via the settings
![image](https://github.com/Acly/krita-ai-diffusion/assets/150020319/bbdb16d4-9365-40e5-87eb-794b4043273a)

**Reply by nothingness6** [reply]
Hi, thanks for the answer. I didn't realize it was so easy. I feel I was stupid. LOL.
Btw, there is an error. IP Adapter Plus is already installed, but the error message says that it is missing. 
Please check the screenshots:
![Krita AI-01](https://github.com/Acly/krita-ai-diffusion/assets/105210258/9d5644af-244b-44da-9e28-33c5c98cecf5)
![Krita AI-02](https://github.com/Acly/krita-ai-diffusion/assets/105210258/8a4fef7a-8780-45e5-8c6f-bec927d9b078)

**Reply by deumiij** [reply]
Hi, is your local comfyui outdated?
You may need to update your comfyui first, since IPAdapter only work with latest comfyui
